upload/catalog/model/extension/shipping/flagship.php

```diff
@@ -111,10 +137,10 @@ protected function getPayload(array $address) : array {
             "is_commercial" => false
         ];
         $packages = [
-            "items" => $this->config->get('shipping_flagship_packing') == 1 ? $this->getPackageItems() : $this->getItems(),
-            "units"=> 'imperial',
-            "type"=>"package",
-            "content"=>"goods"
+            "items" => $this->getPackageItems(),

```

Comment:
We cannot skip this check. There could be people who don't want to use the packing api

Reply:
yes you're right, i'll make the changes right now